### DISCUSSION TITLE
Auto add ID for new resources.

### DISCUSSION THREAD

**Discussion by rufusin** [REAndroid/APKEditor#145] [Q&A]
Tell me, does this utility have an automatic adding of ID for new resources during recompilation? I want to add a new file to the layout folder. During recompilation, it gives an error until I manually add an ID for the new layout file. Am I doing something wrong or is the utility unable to add ID automatically?

**Comment by REAndroid** [comment]
Automatic resource id generation is a complicated process, we have to deal with dex classes, there is ongoing project on [ARSCLib](https://github.com/REAndroid/ARSCLib). 
For now, all I can say is add manually or make some script for your specific task

**Reply by rufusin** [reply]
So the utility can't do this after all? Are you planning to implement this?
